app/src/main/java/com/framgia/moviedb/screen/person/PersonActivity.java

```diff
@@ -0,0 +1,137 @@
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+import com.framgia.moviedb.R;
+import com.framgia.moviedb.data.model.Movie;
+import com.framgia.moviedb.data.model.Person;
+import com.framgia.moviedb.data.repository.PersonRepository;
+import com.framgia.moviedb.data.source.remote.PersonRemoteDataSource;
+import com.framgia.moviedb.screen.more.MoreMovieActivity;
+import com.framgia.moviedb.screen.movie_detail.MovieDetailActivity;
+import com.framgia.moviedb.screen.movie_detail.RelatedMovieAdapter;
+import com.framgia.moviedb.utils.APIUtils;
+
+import java.util.List;
+
+public class PersonActivity extends AppCompatActivity implements PersonContract.View,
+        View.OnClickListener, RelatedMovieAdapter.OnItemClickListener {
+    private static final String EXTRA_PERSON_ID = "EXTRA_PERSON_ID";
+    private static final int FIRST_PAGE_NUMBER = 1;
+    private static final int SIZE_PROFILE_IMAGE = 500;
+    private TextView mTextName;
+    private TextView mTextBirthday;
+    private TextView mTextPlaceOfBirth;
+    private TextView mTextBiography;
+    private ImageView mImageProfile;
+    private RecyclerView mRecyclerView;
+    private int mId;
+
+    public static Intent getIntent(Context context, int personId) {
+        Intent intent = new Intent(context, PersonActivity.class);
+        intent.putExtra(EXTRA_PERSON_ID, personId);
+        return intent;
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_person);
+        initToolbar();
+        getPersonData();
+        initView();
+    }
+
+    @Override
+    public void showInfo(Person person) {
+        mTextName.setText(person.getName());
+        mTextPlaceOfBirth.setText(person.getPlaceOfBirth());
+        mTextBirthday.setText(person.getBirthday());
+        mTextBiography.setText(person.getBiography());
+        String url = APIUtils.PRE_POSTER_URL + person.getProfilePath();
+        Glide.with(this)
+                .load(url)
+                .apply(new RequestOptions()
+                        .placeholder(R.drawable.ic_loading_icon)
+                        .error(R.drawable.ic_error_icon)
+                        .override(SIZE_PROFILE_IMAGE))
+                .into(mImageProfile);
+    }
+
+    @Override
+    public void showInfoError(Exception e) {
+        e.printStackTrace();
+    }
+
+    @Override
+    public void showRelatedMovie(List<Movie> movies) {
+        RelatedMovieAdapter adapter = new RelatedMovieAdapter(this, movies, this);
+        mRecyclerView.setAdapter(adapter);
+        adapter.notifyDataSetChanged();
```

Comment:
trong adapter của recyclerview
method notifyDataSetChanged chỉ gọi khi thật cần thiết 
https://developer.android.com/reference/android/support/v7/widget/RecyclerView.Adapter.html#notifyDataSetChanged()
If you are writing an adapter it will always be more efficient to use the more specific change events if you can. Rely on notifyDataSetChanged() as a last resort.